Containerfile.mcp-template

```diff
@@ -14,8 +14,8 @@ WORKDIR /app
 COPY tracing-config/ ./tracing-config
 
 ARG SERVICE_NAME
-ENV OTEL_SERVICE_NAME=${SERVICE_NAME}
-ENV OTEL_PYTHON_LOGGING_AUTO_INSTRUMENTATION_ENABLED=true
+WORKDIR /app/${SERVICE_NAME}
```

Comment:
In this way we keep directory structure of the code. @tchughesiv was that the idea?